config/coreboot-w541.config

```diff
@@ -603,7 +602,8 @@ CONFIG_SQUELCH_EARLY_SMP=y
 # CONFIG_CONSOLE_NE2K is not set
 CONFIG_CONSOLE_CBMEM=y
 # CONFIG_CONSOLE_CBMEM_DUMP_TO_UART is not set
-# CONFIG_CONSOLE_SPI_FLASH is not set
+CONFIG_CONSOLE_SPI_FLASH=y
+CONFIG_CONSOLE_SPI_FLASH_BUFFER_SIZE=0x20000
```

Comment:
I doubt that 0x20000 would be enough (128kb). 

It would be a compilation testing game + having all logs at the end. We attempted to increase cbmem to 0x1000000 to get 1mb unsuccessfully there since it was not in that region that logs would be added but pre-ram to no effect either.

You can see how much free space is empty on a final build coreboot stitching output either from Ci/local build and adapt space consumed to match until that limit.

@gaspar-ilom that will be useful if that works for general debugging guidelines as well. Document as you go if you can.